### PR TITLE
add GoGuru

### PR DESCRIPTION
**Pull request by alvarolm**
:)

### CODE REVIEW

**Comment by FichteFoll** [comment]
You'll need to change `"branch": "master"` to `"tags": true` and use [tags](http://git-scm.com/book/en/v2/Git-Basics-Tagging) to create releases and push valid [**semantic version**](http://semver.org) tags to your repo. As mentioned [in the docs](https://packagecontrol.io/docs/submitting_a_package#Step_4), we no longer accepts branch-based releases for new packages.

You can import modules relative to your package (in ST3) using a relative import, such as `from .dep import shellenv`. Also note that `dep/shellenv/` contains `.pyc` files, which will be ignored by ST.

show_quick_panel can show entries with multiple lines. Just pass a list of lists of strings instead of a list of strings (thinking of having the description in the second line).

**Comment by alvarolm** [comment]
done :)

File: repository/g.json [hunk 1]
```diff
@@ -880,6 +880,18 @@
...
 		},
 		{
+			"name": "GoGuru",
+			"details": "https://github.com/alvarolm/GoGuru",
+			"labels": ["go", "guru"],
+			"releases": [
+				{
+					"sublime_text": ">=3000",
+					"platforms": ["*"],
+					"branch": "tags"
```
Comment (FichteFoll) [inline repository/g.json]: `"tags": true`, not `"branch": "tags"` ;)

**Comment by FichteFoll** [comment]
What about my second paragraph? The third is not important.

**Comment by alvarolm** [comment]
at the time isnt a critical matter, but it will be implemented **(https://github.com/alvarolm/GoGuru/issues/11),** thanks for the feedback mate .

**Comment by FichteFoll** [comment]
Yes, it is not critical but I didn't want this to go unnoticed, which is why I mentioned it again.

**Pull request merged**